kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/AsyncFetcher.java

```diff
@@ -163,6 +167,21 @@ public Builder<K, V, E> pollTimeout(long timeoutMillis) {
             return this;
         }
 
+        /**
+         * Set the {@code offsetCommitType}, options are:
+         * {@link OffsetCommitType#AUTO} let the Kafka consumer commit offsets automatically in background
+         * {@link OffsetCommitType#COMMIT_SYNC} let the Kafka consumer commit offsets synchronously after processing
+         * {@link OffsetCommitType#COMMIT_ASYNC} let the Kafka consumer commit offsets asynchronously after processing
+         * Defaults to {@code OffsetCommitType#AUTO}
```

Comment:
Minor restructuring suggestion:
```suggestion
     * Sets the {@code offsetCommitType} defining how the {@link FetchEventsTask} will commit offsets during processing of events.
     *
     * Options are:
     * <ul>
     *     <li>{@link OffsetCommitType#AUTO} - let the Kafka consumer commit offsets automatically in background.</li>
     *     <li>{@link OffsetCommitType#COMMIT_SYNC} - let the Kafka consumer commit offsets synchronously after processing.</li>
     *     <li>{@link OffsetCommitType#COMMIT_ASYNC} - let the Kafka consumer commit offsets asynchronously after processing.</li>
     * </ul>
     *
     * Defaults to {@code OffsetCommitType#AUTO}, meaning the offset commit task happens in the background.
```